include/tvm/tir/schedule/schedule.h

```diff
@@ -151,6 +151,18 @@ class ScheduleNode : public runtime::Object {
    * \return The corresponding loop sref
    */
   virtual StmtSRef GetSRef(const LoopRV& loop_rv) const = 0;
+  /*!
+   * \brief Get the block srefs corresponding to an array of BlockRVs
+   * \param block_rvs The BlockRVs to be looked up
+   * \return The corresponding block srefs
+   */
+  virtual Array<StmtSRef> GetSRefs(const Array<BlockRV>& block_rvs) const = 0;
+  /*!
+   * \brief Get the loop srefs corresponding to an array of LoopRVs
+   * \param loop_rvs The LoopRVs to be looked up
+   * \return The corresponding loop srefs
+   */
+  virtual Array<StmtSRef> GetSRefs(const Array<LoopRV>& loop_rvs) const = 0;
```

Comment:
GetSRefs seems like a simple extension of GetSRef with no additional functionalities, so let's remove these two methods from the header.

Reply:
Or just keep a single `GetSRefs<Type>` Template here? Only `GetSRef(const BlockRV& block_rv)` and `GetSRef(const LoopRV& loop_rv)` need to be overwrided.

Reply:
C++ doesn't allow a virtual template function, so I tend to just remove these two methods from header.